core/iwasm/libraries/debug-engine/debug_engine.h

```diff
@@ -36,6 +36,12 @@ typedef struct WASMDebugBreakPoint {
     uint64 orignal_data;
 } WASMDebugBreakPoint;
 
+typedef struct WASMDebugWatchPoint {
+    struct WASMDebugWatchPoint *next;
```

Comment:
```suggestion
    bh_list_link l;
```

Had better use `bh_list_link`

Reply:
Thanks for the feedback, will revise :)